tests/wpt/meta/html/browsers/browsing-the-web/navigating-across-documents/refresh/same-document-refresh.html.ini

```diff
@@ -0,0 +1,3 @@
+[same-document-refresh.html]
+  [Same-Document Referrer from Refresh]
+    expected: FAIL
```

Comment:
Same here

Reply:
I can't really help these ones; I used the logs from the try job to update all the tests with new results, and it can't exclude the known intermittent failures.